kafka_quota.go

```diff
@@ -0,0 +1,79 @@
+package aiven
+
+import "net/url"
+
+type (
+	KafkaQuota struct {
+		User              string `json:"user"`
+		ClientId          string `json:"client-id"`
+		ConsumerByteRate  int    `json:"consumer_byte_rate"`
+		ProducerByteRate  int    `json:"producer_byte_rate"`
+		RequestPercentage int    `json:"request_percentage"`
+	}
+	KafkaQuotasHandler struct {
+		client *Client
+	}
+	DeleteKafkaQuotaRequest struct {
+		User     string `json:"user"`
+		ClientId string `json:"client-id"`
+	}
```

Comment:
Hey, it [looks like](https://api.aiven.io/doc/#tag/Service:_Kafka/operation/ServiceKafkaQuotaDelete) that delete command doesn't take additional options.

Reply:
it doesn't—but it's not what this does, see my review below